.github/workflows/tagger.yml

```
@@ -0,0 +1,53 @@
+name: PR Tagger
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+permissions:
+  pull-requests: write
+
+jobs:
+  tag-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add tags based on changed files
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const tag_map = {
+              'src/torchjd/_autogram/': 'package: autogram',
+              'src/torchjd/_autojac/': 'package: autojac',
+              'src/torchjd/aggregation/': 'package: aggregation',
+              '.github/workflows/': 'CI',
```

```suggestion
              '.github/workflows/': 'ci',
```

I had changed this to `CI` and changed it back just now as it is the type of a conventional commit.